Lombiq.Tests.UI/Pages/OrchardCoreAdminPage.cs

```diff
@@ -1,8 +1,13 @@
 using Atata;
 using Lombiq.Tests.UI.Components;
+using System.Diagnostics.CodeAnalysis;
 
 namespace Lombiq.Tests.UI.Pages;
 
+[SuppressMessage(
+    "Major Code Smell",
+    "S1144:Unused private types or members should be removed",
+    Justification = "Removing private setters result in failing UI Tests with NRE.")]
```

Comment:
For all of these "Removing private setters result in failing UI Tests with NRE." suppressions, figure out why this is the case, not just the symptom. Atata uses them?

Reply:
OK, then please in the justification instead tell that Atata needs this and link to the docs page.